src/components/NewAccountPage/NewAccountPage.jsx

```diff
@@ -100,7 +145,7 @@ export default () => {
                <StyledField
                   required
                   type="text"
-                  placeholder="Last name"
+                  // placeholder="Last name"
                   defaultValue={lastName}
                   onChange={(e) => setLastName(e.target.value)}
                />
```

Comment:
Wrap the Last Name Form.Group into its own Form.Row. This way for now on Desktop it looks a bit nicer.